model-mesh/base/params.env

```diff
@@ -1,7 +1,7 @@
 monitoring-namespace=opendatahub
-odh-mm-rest-proxy=quay.io/opendatahub/rest-proxy:v0.10.0
-odh-modelmesh-runtime-adapter=quay.io/opendatahub/modelmesh-runtime-adapter:v0.11.0-alpha
-odh-modelmesh=quay.io/opendatahub/modelmesh:v0.11.0-alpha
-odh-openvino=quay.io/opendatahub/openvino_model_server:2022.3-gpu
-odh-modelmesh-controller=quay.io/opendatahub/modelmesh-controller:v0.11.0-alpha
-odh-model-controller=quay.io/opendatahub/odh-model-controller:v0.11.0-alpha
+odh-mm-rest-proxy=quay.io/opendatahub/rest-proxy:v0.11.0-rc0
```

Comment:
these should reference the fast tag images right? @Jooho

Reply:
Yes

Reply:
done